test-gui/src/package/view-decoded-linear-position-plot/DecodedLinearPositionPlotView.tsx

```diff
@@ -102,19 +101,28 @@ const DecodedLinearPositionPlotView: FunctionComponent<DecodedLinearPositionProp
         context.imageSmoothingEnabled = false
         context.drawImage(canvas, displayRange[0], 0, displayRange[1] - displayRange[0], canvas.height, 0, 0, panelWidth, panelHeight)
         if (showObservedPositionsOverlay && scaledObservedPositions !== undefined) {
-            const visibleObserved = scaledObservedPositions.slice(downsampledStart, downsampledEnd + 1)
+            const verticalEpsilon = 0.005
```

Comment:
This is a % change for the whole track width, since we previously scaled the observed-position values to the range (0, 1).
We could also use some number of actual canvas pixels as a threshold for discontinuity, but it's probably kind of a wash.